app/src/main/res/layouts/loginActivity/values/button_template_style.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <style name="button_template_style">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:backgroundTint" tools:ignore="NewApi">@color/orange_button</item>
```

Comment:
I'll remove "tools:ignore" attribute there because if you still see the warning it's because you didn't sync the workspace to use API 21. 
And here's instruction:
https://www.geeksforgeeks.org/how-to-change-the-api-sdk-level-in-android-studio/